requirements.txt

```diff
@@ -2,10 +2,11 @@
 colour-runner==0.0.4
 coverage==4.2
 dj-database-url==0.4.1
-django>=1.8,<1.9
+django>=1.10,<1.11
```

Comment:
> `pgcrypto` is required as well.

Ah, quite right -- that's not something that can be installed from PyPI though, as far as I understand it. In fact, I think that's installed with the [migration](https://github.com/incuna/django-pgcrypto-fields/blob/0fe7d8229da7b69787a3e6534b976f256651c00f/pgcrypto/migrations/0001_add_pgcrypto_extension.py) in this project.

> I assumed there was some way to specify a requirement to be met which isn't force installed. I've had a look at the docs and can't see a way to do this.

Yeah, it's a bit of a pain -- I kinda wish there was a way to deal with conflicting requirements. [`pipdeptree`](https://github.com/naiquevin/pipdeptree) can be handy for debugging though. `:)`